receiver/sqlqueryreceiver/integration_test.go

```diff
@@ -125,6 +125,53 @@ func TestPostgresIntegrationLogsTrackingWithoutStorage(t *testing.T) {
 	testAllSimpleLogs(t, consumer.AllLogs())
 }
 
+func TestPostgresIntegrationLogsTrackingByTimestampColumnWithoutStorage(t *testing.T) {
+	// Start Postgres container.
+	externalPort := "15432"
+	dbContainer := startPostgresDbContainer(t, externalPort)
+	defer func() {
+		require.NoError(t, dbContainer.Terminate(context.Background()))
+	}()
+
+	// Start the SQL Query receiver.
+	receiverCreateSettings := receivertest.NewNopSettings()
+	receiver, config, consumer := createTestLogsReceiverForPostgres(t, externalPort, receiverCreateSettings)
+	config.CollectionInterval = 100 * time.Millisecond
+	config.Queries = []sqlquery.Query{
+		{
+			SQL: "select * from simple_logs where insert_time > $1 order by insert_time asc",
+			Logs: []sqlquery.LogsCfg{
+				{
+					BodyColumn:       "body",
+					AttributeColumns: []string{"attribute"},
+				},
+			},
+			TrackingColumn:     "insert_time",
+			TrackingStartValue: "2022-06-03 21:00:00+00",
+		},
+	}
+	host := componenttest.NewNopHost()
+	err := receiver.Start(context.Background(), host)
+	require.NoError(t, err)
+
+	// Verify there's 5 logs received.
+	require.Eventuallyf(
+		t,
+		func() bool {
+			return consumer.LogRecordCount() > 0
+		},
+		1*time.Minute,
+		5*time.Second,
```

Comment:
```suggestion
		100*time.Milliseconds,
```
Minor nit: We don't need to wait 5 seconds between checks here, it may just make the test take longer than necessary.

Reply:
When tick == collection interval (which is also 100ms), problem with records reprocessing may not be detected by this test.
I agree, waiting for 5s is redundant, [I decreased it to 500ms](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/35195/commits/dc1924571d74b9d5a7a3fc0e23792fd5481f2d5e)